### PR TITLE
[WIP] Release candidate v2.2.2

### PR DESCRIPTION
**Pull request by adamrtalbot**
Release candidate v2.2.2

Seems to be some branch protection problems.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/smrnaseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/smrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,11 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v2.2.1](https://github.com/nf-core/smrnaseq/releases/tag/2.2.1) - 2023-05-08
+## Current dev
```
Comment (maxulysse) [inline CHANGELOG.md:6]:
```suggestion
## [v2.2.2](https://github.com/nf-core/smrnaseq/releases/tag/2.2.2) - 2023-06-01```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,11 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v2.2.1](https://github.com/nf-core/smrnaseq/releases/tag/2.2.1) - 2023-05-08
+## [v2.2.2](https://github.com/nf-core/smrnaseq/releases/tag/2.2.2) - 2023-06-01```
```
Comment (maxulysse) [inline CHANGELOG.md:6]:
```suggestion
## [v2.2.2](https://github.com/nf-core/smrnaseq/releases/tag/2.2.2) - 2023-06-01
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ jobs:
     strategy:
       matrix:
         NXF_VER:
-          - "23.04.1"
+          - "22.10.1"
```
Comment (maxulysse) [inline .github/workflows/ci.yml]: why reverting to `22.10.1`?

**Comment by apeltzer** [comment]
Going to close this one as there was also a new template update etc pp.

**Pull request closed without merging**